docs/conflict_resolution.md

```diff
@@ -37,43 +37,18 @@ class UserConflictResolutionHandler(val context: Context) : ConflictResolutionIn
             /* You can run a switch case on the operation type to detect which type of mutation is it in which conflict occured
                and accordingly create an object of that mutation, resolve conflict and make a server call with it.
              */
-            when (operationType) {
-                /*
-                According to the schema structure, used a version based approach of resolving conflicts.
-                If operationType is "UpdateTaskMutation" perform the following steps to resolve conflicts.
-                 */
+            when (operationType) {           
                 "UpdateTaskMutation" -> {
-                 
-                    val input = TaskInput.builder().title(clientState["title"].toString()).version(versionAfterConflict)
-                        .description(clientState["description"].toString()).status("test").build()
-                   
-                    var mutation = UpdateTaskMutation.builder().id(clientState["id"].toString()).input(input).build()
-                   
-                    val mutationCall = Utils.getApolloClient(context)?.mutate(mutation)
-
-                    val callback = object : ApolloCall.Callback<UpdateTaskMutation.Data>() {
-                        override fun onFailure(e: ApolloException) {                       
-                            e.printStackTrace()
-                        }
-
-                        override fun onResponse(response: Response<UpdateTaskMutation.Data>) {
-                            Log.e("onResponse() updateTask", "${response.data()?.updateTask()?.title()}")
-                            val result = response.data()?.updateTask()
-                            //In case of conflicts, data returned from the server is null.
-                            result?.let {
-                                Log.e(TAG, "onResponse-UpdateTask- $it")
-                            }
-                        }
-                    }
-                    mutationCall?.enqueue(callback)
-                }
-                
-                "UpdateUserMutation" -> {
-                    /* Similar for UpdateUserMutation
-                    1. Get all the fields from the clientState.
+                    /* 
+                    1. Get the necessary fields from the clientState.
```

Comment:
Kinda liked that we had some practical example. Maybe we can link to sample app for more practical one.

Reply:
Did you look at the earlier example? Was it okay? (the version based one?)

Reply:
my bad. Yes it is duplicated

Reply:
my bad. Yes it is duplicated